<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XmlConsole</class>
 <widget class="QWidget" name="XmlConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xml stream console</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="filterButton">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="xmlBrowser"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save log</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
